tests/test_paperqa.py

```diff
@@ -463,10 +467,12 @@ def accum(x) -> None:
         outputs.append(x)
 
     llm = anthropic_settings.get_llm()
-    completion = await llm.run_prompt(
-        prompt="The {animal} says",
-        data={"animal": "duck"},
-        system_prompt=None,
+    prompt = "The {animal} says"
+    messages = [
+        Message(role="user", content=prompt.format(animal="duck")),
+    ]
```

Comment:
```suggestion
    messages = [
        Message(role="user", content="The duck says"),
    ]
```

Can we not just do this?